### ISSUE TITLE
TunnelBear + AFWall+ = AFWall+ startup error where it's unable to apply rules.

### ISSUE THREAD

**Issue by code-issues-pull** [ukanth/afwall#468]
Android 5.1.1. Nexus6. No experimental settings are being used.

Install AFWall+, set up your rules how you like them. Install TunnelBear, make rules for it too.

Restart phone.

AFWall generates an error at startup saying it is unable to apply rules.

Enabling AFWall manually works fine. Rules apply fine.

Uninstalling TunnelBear resolves startup rule issues.

Have contacted TunnelBear support about this issue as well just before I typed this out. Letting both sides know in hopes that one is able to resolve it.

**Comment by code-issues-pull** [comment]
Further testing shows that the error on startup is erratic and appearing random. Seems about every other restart it shows up as fine. Attempting to narrow down the issue further now. I only noticed this after installing TunnelBear hence my thought was that was the issue. It may still be, or it may not now since the error randomly appears without it being installed now.

It's not a host file issue, not a custom script issue, happens with either busybox version selected.

So far having wifi disabled at startup (ie. it was off prior to the phone being restarted), resolves the issue.

But if wifi is on, error. The last change I did to wifi was tell it 2.4Ghz networks only. Beyond that, having fucked with anything wifi related. Will test more.
